lib/NeuraDialect/Transforms/GenerateCodePass.cpp

```diff
@@ -80,10 +80,22 @@ struct TileLocation {
 // ---- Operation kind helpers ----.
 static bool isDataMov(Operation *op) { return dyn_cast<DataMovOp>(op) != nullptr; }
 static bool isCtrlMov(Operation *op) { return dyn_cast<CtrlMovOp>(op) != nullptr; }
-static bool isPhi(Operation *op) { return dyn_cast<PhiOp>(op) != nullptr; }
+static bool isPhiLike(Operation *op) { return dyn_cast<PhiStartOp>(op) != nullptr; }
```

Comment:
Why not just name this func as `isPhiStart()`, if it only checks `dyn_cast<PhiStartOp>`?